burst/filtering.py

```diff
@@ -172,12 +172,14 @@ def use_general(self, provider, payload):
             provider (str): Provider ID
             payload (dict): Elementum search payload
         """
+        # Domain name preferred order:
+        # root_url -> base_url (see update_definitions()) -> public_dns_alias OR tor_dns_alias -> user defined alias
         definition = definitions[provider]
-        definition = get_alias(definition, get_setting("%s_alias" % provider))
```

Comment:
I think the idea of using _only_ alias is that is you try to use usual domain and it is firewalled - then you will need to wait for timeout to try another domain. So most probably you would not even try a second domain before Burst will timeout the search.